lib/rules/stylelint-disable-no-empty/index.js

```diff
@@ -0,0 +1,54 @@
+"use strict"
+
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+
+const ruleName = "stylelint-disable-no-empty"
+
+const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected empty `stylelint-disable` comment",
+})
+
+const stylelintDisableCommand = "stylelint-disable"
+
+const rule = (actual) => {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) {
+      return
+    }
+
+    root.walkComments((comment) => {
+      if (!isDisableCommand(comment.text)) {
+        return
+      }
+
+      const specificRules = comment.text.split(" ", 2).slice(1)
```

Comment:
> I don't think I can validate if the rule itself is correct

🤔  ... without that, I'm not sure this rule has a lot of value. If `stylelint-disable .` passes, it's not doing much to prevent abuse.